app/src/main/java/ch/epfl/sdp/user/User.kt

```diff
@@ -0,0 +1,6 @@
+package ch.epfl.sdp.user
+
+class User (
+        var name: String = "",
+        var uid: Int = 0
+)
```

Comment:
Does not follow the user as a singleton design choice we made, or if designed to hold the user datas of other participants and not current, then US-8 will need refactoring

Reply:
That's not supposed to be the singleton for the local user it's a general representation of a user for lobbies and so on

Reply:
This is clearly a temporary solution while waiting for US-8. lgtm